<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRegistrationManipulationWidget</class>
 <widget class="QWidget" name="QmitkRegistrationManipulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>317</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Translation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y (Coronal)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x (Sagittal)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>z (Axial)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="slideTransZ">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="slideTransY">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="slideTransX">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="sbTransX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sbTransY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 240)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="sbTransZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 0, 0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rotation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>y (Coronal)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>x (Sagittal)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>z (Axial)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="slideRotZ">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="slideRotY">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="slideRotX">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="sbRotX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 170, 0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sbRotY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 240)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="sbRotZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 0, 0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupScale">
     <property name="title">
      <string>Scaling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>y (Sagittal)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>x (Frontal)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>z (Axial)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="slideScaleZ">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="slideScaleY">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="slideScaleX">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="sbScaleX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sbScaleY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="sbScaleZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
